src/debug/Benchmark.cpp

```diff
@@ -12,12 +12,12 @@ BenchmarkGuard::BenchmarkGuard(const QString &_name)
 BenchmarkGuard::~BenchmarkGuard()
 {
     qCDebug(chatterinoBenchmark)
-        << this->name_ << float(timer_.nsecsElapsed()) / 1000000.0f << "ms";
+        << this->name_ << float(timer_.nsecsElapsed()) / 1000.0f << "ms";
```

Comment:
This seems incorrect to me, 1 millisecond is 1000000 nanoseconds, not 1000

Reply:
Without your change:
![image](https://user-images.githubusercontent.com/962989/125922246-1fc3300b-6487-4615-82a5-3489b4bed122.png)
![image](https://user-images.githubusercontent.com/962989/125922267-8999e21e-a2fc-404a-a37f-eea2e5044b4a.png)

So I imagine your change would be incorrect, feel free to test it yourself too

Reply:
I've also previously mixed these up but it's milli (10^3), micro (10^6) and then nano (10^9).

Reply:
I must've gotten confused with microseconds. Sorry for this silly change.